src/aws_signature.erl

```diff
@@ -94,6 +94,54 @@ sign_v4(AccessKeyID, SecretAccessKey, Region, Service, DateTime, Method, URL, He
 
     add_authorization_header(FinalHeaders, Authorization).
 
+%% @doc Signs an AWS Event Stream message and returns the headers and
+%% signature used for next event signing.
+%%
+%% Headers of a sigv4 signed event message only contains 2 headers
+%%   * ':chunk-signature'
+%%     * computed signature of the event, binary string, 'bytes' type
+%%   * ':date'
+%%     * millisecond since epoch, 'timestamp' type
```

Comment:
A markdown list won't work here, we need to do same as here:

https://github.com/aws-beam/aws_signature/blob/a40e674f9681f70a6bc5d75035dd3edf7372c8c5/src/aws_signature.erl#L177-L184

Reply:
I updated the docs!